nx/include/switch/applets/error.h

```diff
@@ -0,0 +1,74 @@
+/**
+ * @file error.h
+ * @brief Wrapper for using the error LibraryApplet.
+ * @author StuntHacks
+ * @copyright libnx Authors
+ */
+#pragma once
+#include "../types.h"
+#include "../services/applet.h"
+
+/// Error configuration struct.
+typedef struct {
+    u32 major_code;             ///< First part of the error code.
+    u32 minor_code;             ///< Second part of the error code.
+    bool custom_text;           ///< Whether to display a custom error message.
+    char* short_description;    ///< Short description of the error.
```

Comment:
This is *very* wonky, and unsafe (especially since you're using strcpy). Why not just embed the `u8 args[0x1018]` buffer inside ErrorConfig and have functions fill that buffer?

Reply:
See 7e42dfe (also f816ac5).